.moai/specs/SPEC-GATE-001/acceptance.md

```
@@ -0,0 +1,64 @@
+**Then** return Decision "deny" with the lint error output
+
+## AC-GATE-004: Go Test Gate
+**Given** quality gate is enabled, go vet passes, lint passes, and skip_tests is false
+**When** `go test ./...` fails with exit code != 0
+**Then** return Decision "deny" with the test failure output
+
+## AC-GATE-005: Sequential Execution
+**Given** go vet fails
+**When** the quality gate processes the commit
+**Then** golangci-lint and go test SHALL NOT be executed
+
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)